src/bn256/fq.rs

```diff
@@ -83,7 +83,6 @@ const TWO_INV: Fq = Fq::from_raw([
     0x183227397098d014,
 ]);
 
-// TODO: Can we simply put 0 here::
 const ROOT_OF_UNITY: Fq = Fq::zero();
```

Comment:
I don't think this is correct. We should actually look for the correct root. (I belive we already saw in Vietnam there's only 1).

Reply:
Updated! @CPerezz can we try re-running the tests? They all pass locally.